spyder/plugins/editor.py

```diff
@@ -1929,7 +1929,7 @@ def print_file(self):
                           header_font=self.get_plugin_font('printer_header'))
         printDialog = QPrintDialog(printer, editor)
         if editor.has_selected_text():
-            printDialog.addEnabledOption(QAbstractPrintDialog.PrintSelection)
+            printDialog.setOption(QAbstractPrintDialog.PrintSelection, True)

```

Comment:
@dalthviz can we check that it will work on Qt4 and Qt5, and try to add a small test for it?